### PR TITLE
Add WWGR to summary 

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**

### CODE REVIEW

**Comment by goncalvesmachadoc** [comment]
jenkins build this please

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -1792,6 +1792,7 @@ static const std::unordered_map< std::string, ofun > funs = {
     { "GWCT", div( rate< rt::wat, producer >,
                    sum( rate< rt::wat, producer >, rate< rt::oil, producer > ) ) },
     { "WGOR", div( rate< rt::gas, producer >, rate< rt::oil, producer > ) },
+    { "WWGR", div( rate< rt::wat, producer >, rate< rt::gas, producer > ) },
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:1795]: I think this will be zero for injectors regardless of injected phase, right?  If so, then this is fine.

**Review by bska — COMMENTED** [review]
Thanks.  This looks good to me and once we've clarified the details concerning injectors I'll update the reference solutions and get this into the master branch.

File: src/opm/output/eclipse/Summary.cpp (hunk 1 above)
Comment (goncalvesmachadoc) [reply src/opm/output/eclipse/Summary.cpp:1795]: image

**Comment by bska** [comment]
Thanks, @goncalvesmachadoc. Much appreciated.  I'll update the reference solutions and get this into the master branch.

**Comment by bska** [comment]
jenkins build this update_data please

**Comment by bska** [comment]
jenkins build this opm-tests=953 please

**Comment by bska** [comment]
The new reference solutions have been installed on the CI system.  Merging this now to bring the overall system back to a consistent state.

**Pull request merged**